doc/footer.html

```diff
@@ -17,5 +17,126 @@
 </a> $doxygenversion
 </small></address>
 <!--END !GENERATE_TREEVIEW-->
+<script type="text/javascript">
+//<![CDATA[
+function myStringCompare(str1, str2) {
+   var value = -1;
+   var str1_tmp_char;
+   var str2_tmp_char;
+   var smaller = str1.length;
+   var int1;
+   var int2;
+   if(str2.length == smaller){
+      value = 0;
+   }else if(str2.length < smaller){
+      smaller = str2.length;
+      value = 1;
+   }
+   for(var i=0; i<smaller; i++){
+      str1_tmp_char = str1.charAt(i);
+      str2_tmp_char = str2.charAt(i);
+      if(isNaN(str1_tmp_char) || isNaN(str2_tmp_char)){
+         if(str1_tmp_char < str2_tmp_char){
+            return -1;
+         }else if(str1_tmp_char > str2_tmp_char){
+            return 1;
+         }
+      }else{
+         int1 = parseInt(str1.substring(i));
+         int2 = parseInt(str2.substring(i));
+         if(int1 < int2){
+            return -1;
+         }else if(int1 > int2){
+            return 1;
+         }
+      }
+   }
+   return value;
+}
+function alphabeticOrderElements(elements) {
```

Comment:
so? It's juts that it could be less/faster code (have myStringCompare extract the innerHTML to keep things simple).